proto/tendermint/types/params.proto

```diff
@@ -11,10 +11,10 @@ option (gogoproto.equal_all) = true;
 // ConsensusParams contains consensus critical parameters that determine the
 // validity of blocks.
 message ConsensusParams {
-  BlockParams     block     = 1 [(gogoproto.nullable) = false];
```

Comment:
Is the "nullable" attribute really not needed?

Reply:
This is how it is in v0.35.x and v0.36.x